### DISCUSSION TITLE
missing Interface HasLabel in vaadin 14.9.x

### DISCUSSION THREAD

**Discussion by MHaase70** [vaadin/flow#15373] [General]
Hello,

we use vaadin 14.8.20 and would like to update to vaadin 14.9.2. The maven build no shows an Error:

cannot access com.vaadin.flow.component.HasLabel

I cannot find this class in vaadin 14.9.2, I can find other Interfaces, like HasHelper.

best reegards
Martin

**Comment by MHaase70** [comment]
I opened an issue https://github.com/vaadin/flow/issues/15378

**Comment by mshabarov** [comment]
Closed, as #15378 has been closed.
